docs/scripts/basic.md

```diff
@@ -0,0 +1,70 @@
+## Welcome
+
+**Facilitator**
+Welcome everyone. Thank you so much for taking the time join us together so that we can all have a more satisfying 
+democracy.
+
+## Introductions
+
+**Facilitator**
+Let's begin by sharing our names and pronouns. Also, what brought you here. I'll start first.
+
+My name is Ben. The pronouns I use are "he", "him" and "his" as well as "they", "them" and "theirs". What brings me here is the desire to see a world that works for everyone where we can all take each other seriously.
```

Comment:
I don't necessarily like "where we can all take each other seriously".    Possibly "a world where everyone may express themselves and be treated with respect regardless of opinion or stature"  ?